ChatApp/Custom+UI/ChatView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ChatView.swift
+//  ChatApp
+//
+//  Created by bakhva  on 29.04.23.
+//
+
+import UIKit
+
+final class ChatView: UIView {
+    
+    // MARK: Text Field View
```

Comment:
the name is self-explanatory and does not require // MARK: 

Try grouping Outlets with just 1 mark, like this:
```suggestion
    // MARK: - Outlets
```